Dockerfile_k8s

```diff
@@ -0,0 +1,19 @@
+FROM registry.suse.com/bci/python:3.10
+
+RUN zypper -n in gcc tar gzip kubernetes1.23-client
+
+# Goole cli installation
+RUN cd /opt && curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-415.0.0-linux-x86_64.tar.gz && tar -xf google-cloud-cli-415.0.0-linux-x86_64.tar.gz && rm -f google-cloud-cli-415.0.0-linux-x86_64.tar.gz && google-cloud-sdk/bin/gcloud components install gke-gcloud-auth-plugin
+
+# Install python dependences
+COPY requirements_k8s.txt /pcw/
+RUN  pip3.10 install -r /pcw/requirements_k8s.txt && rm -rf /var/cache
```

Comment:
```suggestion
RUN  pip3.10 install -r /pcw/requirements_test.txt && rm -rf /var/cache
```

because it is **devel** container I think it is big chance that you would like to run tests on your newly created code